pkg/function/client/client.go

```diff
@@ -18,18 +19,17 @@ type client struct {
 }
 
 // New creates a new client object.
-func New(inputOptions ...Option) (*client, error) {
-
-	var opts = &options{
-		sockAddr: function.Addr,
+func New(inputOptions ...sharedutils.Option) (*client, error) {
```

Comment:
Do not touch the `Option` in other packages, let them separated, even there's duplication.

Reply:
The grpc client/server implementation is very similar across UDSink/UDFunction/UDTransformer. The `Options` struct is exactly same. Do we see customization for future UDxxx?